test/e2e/page-objects/identity/appellantDetails.js

```diff
@@ -2,61 +2,80 @@ const appellant = require('test/e2e/data.en').appellant;
 const config = require('config');
 const postcodeLookupContentEn = require('components/postcodeLookup/content.en');
 const postcodeLookupContentCy = require('components/postcodeLookup/content.cy');
+const appellantNameContentEn = require('steps/identity/appellant-name/content.en');
+const appellantNameContentCy = require('steps/identity/appellant-name/content.cy');
+const appellantDOBContentEn = require('steps/identity/appellant-dob/content.en');
+const appellantDOBContentCy = require('steps/identity/appellant-dob/content.cy');
+const appellantNINOContentEn = require('steps/identity/appellant-nino/content.en');
+const appellantNINOContentCy = require('steps/identity/appellant-nino/content.cy');
+
 
 const postcodeLookupEnabled = config.get('postcodeLookup.enabled') === 'true';
 
-function enterAppellantNameAndContinue(commonContent, title, firstName, lastName) {
+function enterAppellantNameAndContinue(language, commonContent, title, firstName, lastName) {
   const I = this;
+  const appellantNameContent = language === 'en' ? appellantNameContentEn : appellantNameContentCy;
 
-  I.wait(1);
+  I.waitForText(appellantNameContent.title.withoutAppointee);
+  // I.waitForElement('#firstName', 3);
   I.selectOption({ id: 'title' }, title);
   I.fillField({ id: 'firstName' }, firstName);
   I.fillField({ id: 'lastName' }, lastName);
   I.click(commonContent.continue);
 }
 
-function enterAppellantNameAndContinueAfterSignIn(commonContent, title, firstName, lastName) {
+function enterAppellantNameAndContinueAfterSignIn(language, commonContent, title, firstName, lastName) {
   const I = this;
+  const appellantNameContent = language === 'en' ? appellantNameContentEn : appellantNameContentCy;
 
-  I.wait(1);
+  I.waitForText(appellantNameContent.title.withoutAppointee);
+  // I.waitForElement('#firstName', 3);
   I.selectOption({ id: 'title' }, title);
   I.fillField({ id: 'firstName' }, firstName);
   I.fillField({ id: 'lastName' }, lastName);
   I.click(commonContent.saveAndContinue);
 }
 
-function enterAppellantDOBAndContinue(commonContent, day, month, year) {
+function enterAppellantDOBAndContinue(language, commonContent, day, month, year) {
   const I = this;
+  const appellantDOBContent = language === 'en' ? appellantDOBContentEn : appellantDOBContentCy;
 
-  I.wait(1);
+  I.waitForText(appellantDOBContent.title.withoutAppointee);
+  // I.waitForElement('input[name*="day"]', 3);
```

Comment:
we can remove commented code unless it is temporary ?

Reply:
@nitinprabhuhmcts  done